regression/contracts/assigns_enforce_havoc_object/functions.txt

```diff
@@ -0,0 +1 @@
+Reading GOTO program from '__CPROVER_file_local_s2n_hash_c_s2n_evp_hash_reset_with_contracts_harness.c.goto'
```

Comment:
What is the meaning of this file?

Reply:
this is a debugging session leftover, removing it